src/main/java/graphql/language/ArrayValue.java

```diff
@@ -87,7 +91,8 @@ public String toString() {
 
     @Override
     public ArrayValue deepCopy() {
-        return new ArrayValue(deepCopy(values), getSourceLocation(), getComments(), getIgnoredChars(), getAdditionalData());
+        List<Value> copiedValues = deepCopy(values);
+        return new ArrayValue(copiedValues != null ? copiedValues : emptyList(), getSourceLocation(), getComments(), getIgnoredChars(), getAdditionalData());
```

Comment:
Want to call out this is a small change to not allow a null value. A list will be used instead. Without this change, there is a NullAway error